### ISSUE TITLE
Take advantage of the Chrome "control center" for unsupported websites

### ISSUE THREAD

**Issue by awsms** [web-scrobbler/web-scrobbler#3159]
https://support.google.com/chrome/answer/9692215

I don't know how exactly it's called, but the "Control your music, videos, and more" button on chrom(ium) browsers automatically detect the Artist/Track/Album metadata.

For example here, it's detecting all of the metadata on a local instance of subsonic
![Screenshot from 2022-03-05 20-41-35](https://user-images.githubusercontent.com/48278661/156897861-334cf2ef-d7ef-4a91-8853-337c9d47cba4.png)

I was wondering if this could be used by extensions API so web-scrobbler could use this information to scrobble tracks, despite being from non-configured websites.

**Comment by inverse** [comment]
This is called the [Media Session API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API) and we already have support for it and is being leveraged on some connectors. They just need to enable `Connector.useMediaSessionApi();` in order for this to be leveraged for scraping meta data.

To achieve what you're asking would require us to inject some generic connector that would attempt to hook into this this API.

**Comment by awsms** [comment]
> This is called the [Media Session API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API) and we already have support for it and is being leveraged on some connectors. They just need to enable `Connector.useMediaSessionApi();` in order for this to be leveraged for scraping meta data.

Thx for this useful info!

**Comment by inverse** [comment]
I don't think we should close it just yet :) it could be useful to explore what we can do further with this.
